datapackage.json

```diff
@@ -1,87 +1,57 @@
 {
-  "name": "world-cities",
-  "title": "Major cities of the world",
+  "bytes": 1019186,
+  "count_of_rows": 26467,
   "description": "List of the world's major cities (above 15,000 inhabitants)",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/datasets/world-cities.git"
-  },
-  "sources": [
-    {
-      "name": "Geonames",
-      "path": "http://www.geonames.org/",
-      "title": "Geonames"
-    }
-  ],
-  "keywords": [
-    "geodata",
-    "city"
-  ],
-  "licensces": [
```

Comment:
Please keep the license and i can see that there was a typo previously.

Reply:
Kept previous version of datapackage.json